libraries/consul_installation_binary.rb

```diff
@@ -86,6 +86,7 @@ def self.binary_basename(node, resource)
         case node['kernel']['machine']
         when 'x86_64', 'amd64' then ['consul', resource.version, node['os'], 'amd64'].join('_')
         when /i\d86/ then ['consul', resource.version, node['os'], '386'].join('_')
+        when /^arm/ then ['consul', resource.version, node['os'], 'arm'].join('_')
```

Comment:
> Also it has to be a regex since there are different arm versions but they all start with arm…

Ah, then it looks good to me 👍 
Yes, you've answered my question. Thank you!